modules/rhcos-install-iscsi-ibft.adoc

```diff
@@ -0,0 +1,58 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_bare_metal/installing-bare-metal.adoc
+// * installing/installing_bare_metal/installing-bare-metal-network-customizations.adoc
+// * installing/installing_bare_metal/installing-restricted-networks-bare-metal.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="rhcos-install-iscsi-ibft_{context}"]
+= Installing {op-system} on an iSCSI boot device using iBFT
+
+On a completely diskless machine, the iSCSI target and initiator values can be passed through iBFT. iSCSI multipathing is also supported.
+
+.Prerequisites
+. You are in the {op-system} live environment.
+. You have an iSCSI target you want to install {op-system} on.
+. Optional: you have multipathed your iSCSI target.
+
+.Procedure
+
+. Mount the iSCSI target from the live environment:
++
+[source,text]
+----
+$ iscsiadm \
+    --mode discovery \
+    --type sendtargets
+    --portal 192.168.0.10 \ <1>
+    --login
+----
+<1> The IP address of the target portal.
+
+. Optional: enable multipathing and start the daemon:
++
+[source,text]
+----
+$ mpathconf --enable && systemctl start multipathd.service
+----
+
+. Install {op-system} onto the iSCSI target using the necessary kernel arguments, specifying `rd.iscsi.firmware=1`:
++
+[source,text]
+----
+$ coreos-installer install \
+    /dev/disk/by-path/<IP_address>:<port>-iscsi-<target_iqn>-lun-<lun> \
+    --append-karg rd.iscsi.firmware=1 \ <1>
+    --append-karg rd.multipath=default \ <2>
+    --console ttyS0 \
+    --ignition-file <path_to_file>
```

Comment:
Looks like there is a series of user-replaceable values here that need footnotes or a
"where"
series
<IP_address>:-iscsi-<target_iqn>-lun-
rd.iscsi.firmware=<1> # automatic or user value?
rd.multipath=<default> # automatic or user value?
<path_to_file>

Since these modules are together, I would try to make the command explanation as twinsies as possible.

Reply:
the stuff with `<>` are user values that vary based on the users preexisting setup, so it's kind of out of scope for the procedure to document that stuff more than like "here's what it is, you need it". i can link to the dracut page where these are documented though